data/hai/unfettered 1 alphas.txt

```diff
@@ -0,0 +1,1532 @@
+		dialog `This doesn't look like a good hiding spot...`
+	on enter "Ferukistek"
+		dialog `There are traces here indeed. You spot one system with no hyperspace links north-east of here. It seems like a logical place for the Alphas to go, considering the drones do not appear to have jump drives.`
+	on enter "Host"
+		dialog
+			`You arrive in the system expecting to find a vault and the ship repairing, but it looks more like... It's a trap!`
+			`	The Auxiliary from earlier is indeed here, but it's not damaged at all, and it looks like it's waiting for you with quite a few drones, more than last time.`
+			`	Whatever the case, it probably has the jump drives on board or on the nearby planet. You should first capture the ship, as asked by the Unfettered, and then inspect the planet. Strangely, you do not find a single trace of the particle trace you have been following so far in this system.`
+	npc capture
+		government "Cerberus"
+		system "Host"
+		personality heroic nemesis staying vindictive target
+		ship "Auxiliary (Alpha)" "Settler"
+		conversation
+			action
+				set "unfettered: captured the auxiliary"
+			`While capturing the ship, you noticed that only very few of the crew were actually Alphas. What little were Alphas seem to have disappeared in the middle of the fight. They probably used an escape pod to get away unnoticed, or maybe they were never truly there in the first place. Whatever the case, you cannot find any trace of them. The crew was composed of a mix of mercenaries and indoctrinated pirates that would obey only to 'the Masters' and did not reveal anything useful.`
+			`	There was no doubt who was in command of this vessel, however. The jump drives don't seem to be aboard the ship. You should head to the Alpha base to see if you can find them or anything else of interest.`
+	npc
+		government "Cerberus"
+		system "Host"
+		personality heroic nemesis staying
+		ship "Far Osk 27" "Apathy"
+		ship "Far Osk 27" "Vanity"
+		ship "Far Osk 27" "Despair"
+		ship "Far Osk 27" "Envy"
+		ship "Far Osk 27" "Lust"
+		ship "Far Osk 27" "Pride"
+	# additional fighters if the Alphas were uncontested by Midas. one can use these to fill their Auxiliary with fighters
+	npc
+		government "Cerberus"
+		system "Host"
+		personality heroic nemesis staying
+		to spawn
+			has "midas: wronged"
+		ship "Far Osk 27" "Wrath"
+		ship "Far Osk 27" "Gluttony"
+		ship "Far Osk 27" "Greed"
+		ship "Far Osk 27" "Desecration"
+	to fail
+		has "Wanderers: Defend Vara Ke'sok: done"
+	to fail
+		has "unfettered: outcast"
+	on fail
+		dialog phrase "Essential Unfettered Mission Fail"
+	on visit
+		dialog `You have landed, but the Auxiliary is not your flagship, or you landed without the jump drives. Go and look for traces of fuel from the Auxiliary in systems neighboring Hi Yahr. Don't forget to land on the Alpha base to recover any information or jump drives it may have! Don't forget the Jump Drives need to be on your flagship or in cargo.`
```

Comment:
I feel another capitalization debate coming here…

Reply:
uh I used JD uncapitalized and then capitalized
I wish I knew when to use which haha

Reply:
So do I. If someone would establish a general guideline that would be very helpful. **sigh** @tibetiroka ?

Reply:
I think there was a PR to standardize it

Reply:
It should be uncapitalized.